themes/scss/common/shapes.scss

```diff
@@ -160,3 +160,32 @@
     bottom: -6px;
   }
 }
+
+.CDB-Shape-magnify {
+  display: block;
+  width: 32px;
+  height: 32px;
+  position: relative;
+  &:before{
+    display: block;
+    position: absolute;
+    top: 10px;
+    left: 10px;
+    width: 7px;
+    height: 7px;
+    border: 1px solid #636D72;
+    border-radius: 50%;
+    content: '';
+  }
+  &:after{
+    display: block;
+    position: absolute;
+    top: 17px;
+    left: 19px;
+    width: 1px;
+    height: 6px;
+    background: #636D72;
```

Comment:
Properties should be ordered display, position, top, left, width, height, transform, background, content